README.md

```diff
@@ -215,6 +223,245 @@ objectives:
+the alert will be triggered. Two or more condition can be logically joined using
+the logical operators `AND` or `OR`.
+
+The result of the alert conditions is either `true`, `false` or `indeterminate`.
+
+The table below lists the outcomes of the different combinations:
+
+| Expression                      | Outcome       |
+|---------------------------------|---------------|
+| true AND true                   | true          |
+| true AND false                  | false         |
+| false AND false                 | false         |
+| true AND indeterminate          | indeterminate |
+| indeterminate AND indeterminate | indeterminate |
+| true OR true                    | true          |
+| true OR false                   | true          |
+| false OR false                  | false         |
+| true OR indeterminate           | true          |
+| indeterminate OR indeterminate  | indeterminate |
+| false OR indeterminate          | indeterminate |
+
+In the table below lists when alert policy will be triggering depending on
+the resolved alert condition value:
+
+| Alert condition | alertIf | Alert triggering |
+|---|---|---|
+| true | true | Yes |
+| true | false | No |
+| true | indeterminate | No |
+| false | true | Yes |
+| false | false | No |
+| false | indeterminate | No |
+| indeterminate | true | No |
+| indeterminate | false | No |
+| indeterminate | indeterminate | Yes |
+
+---
+
+#### Alert Condition
+
+An Alert Condition allows you to define in which conditions a alert of SLO
+needs to be triggered.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertCondition
+metadata:
+  name: string
+  displayName: string # optional
+spec:
+  description: # optional
+  severity: string # required (ticket or page)
+  condition: # optional
+    kind: string
+    threshold: number
+    lookbackWindow: number
+    alertAfter: number
+```
+
+#### Notes (Alert Condition)
+
+- **severity** *enum(ticket, page)*, required field. The severity level of the alert
+- **condition**, required field. Defines the conditions of the alert
+  - **kind** *enum(burnrate, custom)* the kind of alerting condition thats checked, defaults to `burnrate`
+
+If the kind is `burnrate` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **lookbackWindow** *number*, required field, the time-frame for which to calculate the threshold
+- **alertAfter** *number*: required field, the duration the condition needs to be valid, defaults `0m`
+
+If the kind is `custom` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **comparison** *enum(lt, lte, gt, gte, eq)*, optional field, defines
+  how the threshold should be compared to meet the threshold, defaults to `gt`
+
+If the alert condition can not be resolved to a `true` or `false` value, for example
+due to missing data points, then the `indeterminate` needs to be returned.
```

Comment:
💡 Should we instead of adding the `indeterminate` have an optional filed `treatMissing` that defaults to something sensible, like CloudwatchAlarms do, see [here] (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#treat_missing_data)